phi/storage/agent/singlestore.py

```diff
@@ -52,43 +56,55 @@ def __init__(
         self.db_engine: Engine = _engine
         self.metadata: MetaData = MetaData(schema=self.schema)
 
+        # Table schema version
+        self.schema_version: int = schema_version
+        # Automatically upgrade schema if True
+        self.auto_upgrade_schema: bool = auto_upgrade_schema
+
         # Database session
         self.Session: sessionmaker[Session] = sessionmaker(bind=self.db_engine)
-
         # Database table for storage
         self.table: Table = self.get_table()
 
-    def get_table(self) -> Table:
+    def get_table_v1(self) -> Table:
         return Table(
             self.table_name,
             self.metadata,
-            # Primary key for this run
-            Column("run_id", mysql.TEXT, primary_key=True),
-            # Assistant name
-            Column("name", mysql.TEXT),
-            # Run name
-            Column("run_name", mysql.TEXT),
-            # ID of the user participating in this run
+            # Session UUID: Primary Key
+            Column("session_id", mysql.TEXT, primary_key=True),
+            # ID of the agent that this session is associated with
+            Column("agent_id", mysql.TEXT),
+            # ID of the user interacting with this agent
             Column("user_id", mysql.TEXT),
-            # -*- LLM data (name, model, etc.)
-            Column("llm", mysql.JSON),
-            # -*- Assistant memory
+            # Agent memory
             Column("memory", mysql.JSON),
-            # Metadata associated with this assistant
-            Column("assistant_data", mysql.JSON),
-            # Metadata associated with this run
-            Column("run_data", mysql.JSON),
-            # Metadata associated with the user participating in this run
+            # Agent Metadata
+            Column("agent_data", mysql.JSON),
+            # User Metadata
             Column("user_data", mysql.JSON),
-            # Metadata associated with the assistant tasks
-            Column("task_data", mysql.JSON),
-            # The timestamp of when this run was created.
-            Column("created_at", DateTime(timezone=True), server_default=text("now()")),
-            # The timestamp of when this run was last updated.
-            Column("updated_at", DateTime(timezone=True), onupdate=text("now()")),
+            # Session Metadata
+            Column("session_data", mysql.JSON),
+            # The Unix timestamp of when this session was created.
+            Column(
+                "created_at",
+                mysql.BIGINT,
+                nullable=False,
+            ),
+            # The Unix timestamp of when this session was last updated.
+            Column(
+                "updated_at",
+                mysql.BIGINT,
+                nullable=False,
```

Comment:
@ysolanky this can be nullable, lets not add these database conditions as they make everything